app/models/route.rb

```diff
@@ -1,9 +1,73 @@
+require 'json'
+
 class Route < ApplicationRecord
   belongs_to :user
   has_many :waypoints, dependent: :destroy
   has_one :contact, through: :user
 
 # default status is 0, active
 # maybe we don't even need this if we are going to destroy upon checking in
-  enum status: {active: 0, complete: 1}
+  enum status: {active: 0, complete: 1, notified: 2}
+
+  def self.send_alerts(time)
+    time = time
```

Comment:
What is the purpose of this line?

Reply:
I think this is a relic of some early debugging and can probably be removed.